script.js

```diff
@@ -1,4 +1,105 @@
+// Base
+// Requirements
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+var gameMode = "dice roll";
+var playerScores = [];
+var playerRolls = [];
+var myOutputValue = "";
+var playerTurn = 1;
+var numPlayers = 0;
+
+// possible gameModes
+// "dice roll", "choose order"
+
+// create function that rolls a random dice number
+var rollDice = function () {
+  // produces a decimal between 0 and 6
+  var randomDecimal = Math.random() * 6;
+  // take off the decimal
+  var randomInteger = Math.floor(randomDecimal);
+  // it's a number from 0 - 5 ... add 1
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// create function that rolls 2 dice for player, stores these numbers in an array and returns a message informing player of the 2 dice numbers
+var playerRoll = function () {
+  playerRolls.push(rollDice());
+  playerRolls.push(rollDice());
+  console.log(`playerRolls: ${playerRolls}`);
+  myOutputValue = `Welcome Player ${playerTurn}. <br> Dice 1: ${playerRolls[0]} <br> Dice 2: ${playerRolls[1]} <br> Please choose the order of the dice.`;
```

Comment:
Instead of defining my OutputValue then returning it you could instead just return the string straight away.
```suggestion
  return `Welcome Player ${playerTurn}. <br> Dice 1: ${playerRolls[0]} <br> Dice 2: ${playerRolls[1]} <br> Please choose the order of the dice.`;
```
Tidies up code and removes unnecessary variables.